build/release.mk

```diff
@@ -21,7 +21,7 @@ release/clean:
 .PHONY : release/deps
 release/deps: $(GORELEASER_BIN)
 	@echo "===> $(INTEGRATION) === [release/deps] install goversioninfo"
-	@go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo
+	@GO111MODULE=off go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo
```

Comment:
this could be tracked in gomodules as well, it's usually done by adding a black import

Reply:
This go get is installing the binary in the GOBIN if we add it as a tool in the go modules, it will only add the src to the gopath, at least I tried it and I got that result.

Reply:
oh I didn't realise this was triggering a binary install, should this be done via `go install` instead?